wporg-gp-translation-events.php

```diff
@@ -162,12 +161,47 @@ public function event_meta_boxes() {
 	 */
 	public function event_dates_meta_box( WP_Post $post ) {
 		wp_nonce_field( 'event_dates_nonce', 'event_dates_nonce' );
-		$event_start = get_post_meta( $post->ID, '_event_start', true );
-		$event_end   = get_post_meta( $post->ID, '_event_end', true );
-		echo '<label for="event_start">Start Date: </label>';
-		echo '<input type="date" id="event_start" name="event_start" value="' . esc_attr( $event_start ) . '" required>';
-		echo '<label for="event_end">End Date: </label>';
-		echo '<input type="date" id="event_end" name="event_end" value="' . esc_attr( $event_end ) . '" required>';
+		$event_start    = get_post_meta( $post->ID, '_event_start', true );
+		$event_end      = get_post_meta( $post->ID, '_event_end', true );
+		$event_timezone = get_post_meta( $post->ID, '_event_timezone', true );
+		$hosts          = explode( ',', get_post_meta( $post->ID, '_hosts', true ) );
+		if ( empty( $hosts ) ) {
+			$hosts = array( $post->post_author );
+		}
+		?>
+		<label for="event_start">Start Date: </label>
+		<input type="datetime-local" id="event_start" name="event_start" value="<?php echo esc_attr( $event_start ); ?>" required><br>
+		<label for="event_end">End Date: </label>
```

Comment:
As we are using UTC times, and we put a timezone selector, to avoid confusions, I suggest adding the `(UTC)` text.

```suggestion
		<label for="event_start">Start Date (UTC): </label>
		<input type="datetime-local" id="event_start" name="event_start" value="<?php echo esc_attr( $event_start ); ?>" required><br>
		<label for="event_end">End Date (UTC): </label>
```